api/v1alpha1/openapi.yaml

```diff
@@ -2547,16 +2802,16 @@ components:
       - 'Synced'               # ResourceSync
       - 'OverlappingSelectors' # Fleet
       - 'Valid'                # Fleet
-      - 'Ready'                # Device
-      - 'DiskPressure'         # Device
-      - 'MemoryPressure'       # Device
-      - 'PIDPressure'          # Device
-      - 'CPUPressure'          # Device
-      - 'Available'            # Device
-      - 'Progressing'          # Device
-      - 'Degraded'             # Device
-      - 'SystemdUnitsRunning'  # Device
-      - 'ContainersRunning'    # Device
+      - 'Ready'                # Device (deprecated)
+      - 'DiskPressure'         # Device (deprecated)
+      - 'MemoryPressure'       # Device (deprecated)
+      - 'PIDPressure'          # Device (deprecated)
+      - 'CPUPressure'          # Device (deprecated)
+      - 'Available'            # Device (deprecated)
+      - 'Progressing'          # Device (deprecated)
+      - 'Degraded'             # Device (deprecated)
+      - 'SystemdUnitsRunning'  # Device (deprecated)
+      - 'ContainersRunning'    # Device (deprecated)
```

Comment:
Should we just go ahead and remove them? may be in a follow up PR if those are used from code let's first agree on the API change.

Reply:
Right, removal will require code.